pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -502,18 +505,18 @@ func modifyOutboundRouteConfig(push *model.PushContext, in *plugin.InputParams,
 				svc := in.Node.SidecarScope.ServiceForHostname(hostname, push.ServiceByHostnameAndNamespace)
 				attrs = addDestinationServiceAttributes(make(attributes), svc)
 			}
-			addFilterConfigToRoute(in, httpRoute, attrs)
-
+			addFilterConfigToRoute(in, httpRoute, attrs, getQuotaSpec(in, hostname, isPolicyCheckDisabled))
```

Comment:
https://github.com/istio/istio/pull/21437#discussion_r384254031

Reply:
@hzxuzhonghu : replied to your comment